docs/WritingTests.asciidoc

```diff
@@ -2333,7 +2333,7 @@ It is also possible to create a GitHub workflow that will clone and monitor an
 openQA job which is mentioned in the PR description or comment. The scripts
```

Comment:
the coment still mentions the repo as `script` here. Not big deal but... could you rename that as well?

Reply:
<img width="1842" height="180" alt="image" src="https://github.com/user-attachments/assets/2570b37f-7dcf-4792-96bf-e4740048afae" />

Actually what *workflow* is this about? I don't see any workflow using openqa-clone-and-monitor-job-from-pr in https://github.com/os-autoinst/os-autoinst-scripts/tree/master/.github/workflows 🤔

Reply:
Ah but wait there is an action https://github.com/os-autoinst/os-autoinst-scripts/blob/master/actions/clone-job/action.yaml